mbuild/utils/conversion.py

```diff
@@ -81,21 +101,22 @@ def OPLS_to_RB(f0, f1, f2, f3, f4, error_tol=1e-4):
     r"""Convert OPLS type to Ryckaert-Bellemans type dihedrals.
 
     .. math::
-    OPLS_torsions &= f0/2 + f1/2*(1+cos(t)) + f2/2*(1-cos(2*t)) + \\
-                  &= f3/2*(1+cos(3*t)) + f4/2*(1-cos(4*t))
+    OPLS_torsions &= \frac{f_0}{2} + \frac{f_1}{2}*(1+cos(t)) + \frac{f_2}{2}*(1-cos(2*t)) + \\
+                  &= \frac{f_3}{2}*(1+cos(3*t)) + \frac{f_4}{2}*(1-cos(4*t))
 
     .. math::
-    RB_torsions &= c0 + c1*cos(psi) + c2*cos(psi)^2 + c3*cos(psi)^3 + \\
-                &= c4*cos(psi)^4 + c5*cos(5*psi)^5
+    RB_{torsions} &= c_0 + c_1*cos(psi) + c_2*cos(psi)^2 + c_3*cos(psi)^3 + \\
+                  &= c4*cos(psi)^4 + c5*cos(5*psi)^5
```

Comment:
same here for c4 and c5

Reply:
Changed for both OPLS_to_RB and RB_to_OPLS